### PR TITLE
feat(CurriculumCard): Add beta sticker option to CurriculumCard

### PR DESCRIPTION
**Pull request by katiezutter**
### Overview

Adds optional beta sticker to CurriculumCards.

Screen Shot 2021-08-09 at 1 28 36 PM

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/cYsS23HjZMH2kY1vrw09Ny/Project-Based-Learning-Lab?node-id=43%3A125
- [x] Related to JIRA ticket: [EGG-1632](https://codecademy.atlassian.net/browse/EGG-1632)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.21-alpha.75bb78.0<br/>@codecademy/gamut-labs@19.19.7-alpha.75bb78.0<br/>@codecademy/styleguide@42.7.6-alpha.75bb78.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61117046be98901af8b0d639--gamut-preview.netlify.app>https://61117046be98901af8b0d639--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61117046be98901af8b0d639>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.21-alpha.83f895.0<br/>@codecademy/gamut-labs@19.19.7-alpha.83f895.0<br/>@codecademy/styleguide@42.7.6-alpha.83f895.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.21-alpha.669ea9.0<br/>@codecademy/gamut-labs@19.19.7-alpha.669ea9.0<br/>@codecademy/gamut@35.14.0-alpha.669ea9.0<br/>@codecademy/styleguide@42.7.6-alpha.669ea9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61117383de851d1b869b2919--gamut-preview.netlify.app>https://61117383de851d1b869b2919--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61117383de851d1b869b2919>Deploy Logs</a></p>

**Comment by katiezutter** [comment]
Who do I ask to make updates to the figma at the top of the storybook page? @songstephen would that be you?

Screen Shot 2021-08-09 at 1 32 51 PM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.22-alpha.3030cf.0<br/>@codecademy/gamut-labs@19.19.8-alpha.3030cf.0<br/>@codecademy/styleguide@42.7.7-alpha.3030cf.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.22-alpha.a670ff.0<br/>@codecademy/gamut-labs@19.19.8-alpha.a670ff.0<br/>@codecademy/styleguide@42.7.7-alpha.a670ff.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://611177fd1bf4862314e6dcf5--gamut-preview.netlify.app>https://611177fd1bf4862314e6dcf5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/611177fd1bf4862314e6dcf5>Deploy Logs</a></p>

**Review by hmafzal — APPROVED** [review]
LGTM!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.24-alpha.e05f51.0<br/>@codecademy/gamut-labs@19.19.10-alpha.e05f51.0<br/>@codecademy/styleguide@43.0.2-alpha.e05f51.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://611424b0037a4a00bc8ae4c1--gamut-preview.netlify.app>https://611424b0037a4a00bc8ae4c1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/611424b0037a4a00bc8ae4c1>Deploy Logs</a></p>

File: packages/gamut-labs/src/CurriculumCard/Footer/index.tsx [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+    if (beta) {
+      return (
+        <Box position="absolute" bottom={0} right={0} pb={16} pr={16}>
+          <BetaSticker />
+        </Box>
+      );
+    }
+    if (tag && tagColor) {
+      return <BottomTag text={tag} color={tagColor} />;
+    }
```
[the remark this answers is not in the document]
Comment (katiezutter) [reply packages/gamut-labs/src/CurriculumCard/Footer/index.tsx:29]: I like yours better - less nesting I think is clearer!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.25-alpha.ca9cf1.0<br/>@codecademy/gamut-labs@19.19.11-alpha.ca9cf1.0<br/>@codecademy/styleguide@43.1.1-alpha.ca9cf1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61154e8ad960870a5ec82924--gamut-preview.netlify.app>https://61154e8ad960870a5ec82924--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61154e8ad960870a5ec82924>Deploy Logs</a></p>

**Pull request merged**